apps/marketing/src/components/(marketing)/footer.tsx

```diff
@@ -81,15 +81,26 @@ export const Footer = ({ className, ...props }: FooterProps) => {
           © {new Date().getFullYear()} Documenso, Inc. All rights reserved.
         </p>
 
-        <div className="flex flex-wrap items-center gap-x-4 gap-y-2.5">
-          <button type="button" className="text-muted-foreground" onClick={() => setTheme('light')}>
-            <Sun className="h-5 w-5" />
-            <span className="sr-only">Light</span>
-          </button>
-
-          <button type="button" className="text-muted-foreground" onClick={() => setTheme('dark')}>
-            <Moon className="h-5 w-5" />
-            <span className="sr-only">Dark</span>
+        <div className="flex flex-wrap">
+          <button
+            type="button"
+            className="text-muted-foreground"
+            onClick={() => {
+              setTheme(theme === 'light' ? 'dark' : 'light');
+            }}
+            aria-label="Theme change"
+          >
+            {theme === 'light' ? (
+              <>
+                <Sun className="h-5 w-5" />
+                <span className="sr-only">Light</span>
+              </>
+            ) : (
+              <>
+                <Moon className="h-5 w-5" />
+                <span className="sr-only">Dark</span>
+              </>
+            )}
           </button>
```

Comment:
@Mythie what do you think of combining the buttons?

Reply:
I think I prefer them individually since it matches the current app, but am open to opinions 😄

Reply:
Just noting our designer is going to look into this, I'll update this once we decide on how to proceed.

Reply:
@adithyaakrishna our designer came back with this:

https://www.figma.com/proto/Qm2heQggjZyygnSQdS22Xa/Design-exploration?type=design&node-id=101-11&t=kwthccpzca53L9vd-1&scaling=scale-down&page-id=0%3A1&starting-point-node-id=101%3A11&show-proto-sidebar=1

If you want to pass on implementing this please let us know so we can release it to the community to implemented

Reply:
@dguyen This looks super cool! I can work on this. Is it fine if I send a fix over the weekend?

Reply:
Yeah that's fine!

Reply:
@dguyen Could you please share the Figma link for the same? the above one seems to be for the prototype. I thought to use the CSS styles from the Figma file itself

Reply:
@adithyaakrishna I currently don't have access to the figma file as our designer is currently on leave

I'll try follow up with this once he's back, but it might take some time (days)

Reply:
Could you please let me know if the above changes looks good?